kubernetes/chart/vault-ui/Chart.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+description: A Helm chart for Kubernetes
+name: vault-ui
+version: 0.1.0
```

Comment:
Chart version and application version are 2 different things. Chart version indicates which set of resources the chart contains. Application version indicates the version of the runtime.

Happy to change the description, but I strongly advise not to sync app and chart version.